setup.py

```diff
@@ -13,6 +13,7 @@
       install_requires=[
           'pipelinewise-singer-python==1.2.0',
           'zenpy==2.0.25',
+          'requests==2.20.0'
```

Comment:
This requirement was included in the https://github.com/twilio-labs/twilio-tap-zendesk (where this repo was forked from)